### PR TITLE
Update Information file

### PR DESCRIPTION
**Pull request by parispittman**
Added preliminary information

@castrojo

### CODE REVIEW

File: community/2017-events/12-dev-summit/ContribSummitInformation.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+The discussion sessions will be in an open fishbowl format — rings of chairs, with inner rings driving the discussion — where anyone can contribute. The various sessions will be proposed and voted on by the community in the weeks leading up to the event. This allows the community to motivate the events of the day without dedicating precious day-of time to choosing the sessions and making a schedule.
+
+There will be 3 rooms dedicated to these sessions running in parallel all day. Each session should last between 45 minutes and an hour.
+
+Then, there will be 2 smaller rooms for hacking / unstructured discussion all day. 
+
+## Call for Topics
+Invites will include an area to list your proposed topic to lead for a potential session. Schedule will be announced at least one week before the event. 
+
+## Desired Outcomes
```
Comment (xiangpengzhao) [inline community/2017-events/12-dev-summit/ContribSummitInformation.md]: Did you intend to set this section to be a list? If so, you should add `- ` (or `* `) at the beginning of each sentence. If not, you should add period at the end of each sentence. :)

Comment (parispittman) [reply community/2017-events/12-dev-summit/ContribSummitInformation.md]: fixed - thank you for pointing it out :)

File: community/2017-events/12-dev-summit/ContribSummitInformation.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+Time TBA
+
+## Where:
+Austin Convention Center
+500 E Cesar Chavez St, 
+Austin, TX 78701
+(same location as KubeCon)
+
+## Invitations:
+Seleted attendees are Kubernetes upstream contributors including SIG members and lottery winners from a members of standing pool.
```
Comment (cblecker) [inline community/2017-events/12-dev-summit/ContribSummitInformation.md]: spelling: Seleted -> Selected

Comment (cblecker) [reply community/2017-events/12-dev-summit/ContribSummitInformation.md]: nit: "members of standing" should probably be quoted so it reads clearer

**Pull request merged**